### ISSUE TITLE
GNU Radio Companion Conda build blocks windows clipboard

### ISSUE THREAD

**Issue by DavidPowell** [conda-forge/gnuradio-feedstock#111]
When running GNU Radio companion from this Conda package under Windows, after a while the Windows clipboard stops working - it is.

I am using GNURadio 3.10.1.0 package from conda-forge, with Mambaforge Python 3.9, on Windows 11. However, I have previously observed this issue with earlier versions of Python on a Windows 10 machine.

I am posting issue here, because I assume it is Windows specific, and related to the packaging of appropriate GUI or system libraries. But please advise if there is a more appropriate place to raise this issue, or if there are any steps I could take to help diagnose this issue.

**Comment by ryanvolz** [comment]
When you say the clipboard stops working, what exactly does that entail? Does it sound anything like this? https://gitlab.gnome.org/GNOME/gtk/-/issues/2215 Supposedly that was fixed, but maybe it is cropping up again in a different form.

It's probably really an issue with GTK, but maybe it's specific to GNU Radio Companion, so this is as good a place as any until we can narrow it down. Have you by chance previously used a non-conda version of GRC and also seen this, so we can know to look at packaging-specific issues or not?

**Comment by DavidPowell** [comment]
I don't think it is the exact GTK issue that you are referring to. I cannot reliably cause the bug just by opening and closing GNU Radio companion. But if I am using it for a while and am using it, at some point I find that cut and paste no longer works.

One possibly relevant factor is that I am using embedded python blocks, which require opening an external text editor (in my case NotePad++). However, performing this action does not seem to directly trigger the bug.

In the past I have used the windows build of Gnu Radio from GNC, and I don't recall this being an issue there - but this was some time ago, and with much older GnuRadio version, so it's not directly comparable.

**Comment by ryanvolz** [comment]
Puzzling. I'll leave GRC open for a while and see if I can observe it, although I may have complications due to running in a VM which connects my host system clipboard to the Windows one.

Questions:

1. Is it perhaps just the keyboard shortcuts that don't work (Ctrl-X, Ctrl-C, Ctrl-V), and maybe right clicking and selecting Copy/Paste still works? Are those right click options even available, or maybe grayed out?
2. Is cut and paste only disabled for GNU Radio Companion (or any GTK application), or is it Windows-wide?
3. How do you restore cut and paste to working? Does it take an application restart or a system restart?

**Comment by DavidPowell** [comment]
The issue is quite intermittent, I have just encountered it again after leaving Gnu Radio companion running for a couple of hours, despite not having experienced it for at least a week.

1. It is not just keyboard shortcuts, the problem also occurs with selecting copy and paste from the menu
2. The issue seems to affect all Windows programs, not just Gnu Radio of Gtk related.
3. This time I was able to fix the problem simply by closing Gnu Radio Companion
